### PR TITLE
Update max txn size limit to 1mb

### PR DESCRIPTION
**Pull request by LukasGasior1**

### CODE REVIEW

File: core/src/test/java/com/radixdlt/rev2/REv2LargeTransactionTest.java [hunk 1]
```diff
@@ -124,9 +124,10 @@ private DeterministicTest createTest() {
   }
 
   private static RawNotarizedTransaction createLargeValidTransaction() {
-    var intentBytes =
+    final var blobsSize = 1024 * 1024 - 462 /* space for other intent/notarized txn fields */;
```
Comment (malexandru-rdx) [inline core/src/test/java/com/radixdlt/rev2/REv2LargeTransactionTest.java:127]: How is `462` calculated? Number of signatures * signature size + some header size?

File: core/src/main/java/com/radixdlt/p2p/transport/PeerChannelInitializer.java [hunk 2]
```diff
@@ -98,7 +98,9 @@ public final class PeerChannelInitializer extends ChannelInitializer<SocketChann
   // This needs to be larger than proposals / vertices
   // See also constant/s in REv2TransactionsAndProofReader
   // TODO - update this number once we have a REP/RNP for these numbers
-  private static final int MAX_PACKET_LENGTH = 25 * 1024 * 1024;
+  // Current max txn size is 1024*1024 and there are max 10 txns in a proposal
+  // + adding an extra mb for any vertex/proposal/qc overhead
+  private static final int MAX_PACKET_LENGTH = 11 * 1024 * 1024;
```
Comment (malexandru-rdx) [inline core/src/main/java/com/radixdlt/p2p/transport/PeerChannelInitializer.java:103]: I'd prefer it to be more verbose/built on top of other limits.
Something along the lines:
```java
private static final int MAX_PROPOSAL_TRANSACTION_COUNT = 10;
private static final int MAX_PROPOSAL_HEADER_SIZE = 1024 * 1024;
private static final int MAX_PROPOSAL_PACKET_SIZE = MAX_PROPOSAL_HEADER_SIZE + MAX_PROPOSAL_TRANSACTION_COUNT * MAX_TRANSACTION_SIZE;
```

But this is out of scope for this PR :).

**Review by malexandru-rdx — APPROVED** [review]

File: core/src/test/java/com/radixdlt/rev2/REv2LargeTransactionTest.java (hunk 1 above)
Comment (LukasGasior1) [reply core/src/test/java/com/radixdlt/rev2/REv2LargeTransactionTest.java:127]: yeah, it's everything else. intent header and signatures
